datacats/error.py

```diff
@@ -4,7 +4,10 @@
 class DatacatsError(Exception):
 
     def __init__(self, message, format_args=(), parent_exception=None):
-        self.message = message
+        try:
+            self.message = message.format(format_args)
+        except (ValueError, KeyError):
+            self.message = message
```

Comment:
this is odd. Why would you be getting Value/KeyErrors it's only our code that creates these.